src/main/java/de/retest/web/selenium/ByRetestId.java

```diff
@@ -0,0 +1,63 @@
+package de.retest.web.selenium;
+
+import java.io.Serializable;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.ui.descriptors.Element;
+import de.retest.ui.descriptors.RootElement;
+import de.retest.ui.diff.Alignment;
+
+public class ByRetestId extends By implements Serializable {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Override the "equals" method in this class. ([squid:S2160](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2160))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=77&issues=AWaHMZUjlB6-zLfzofF-&open=AWaHMZUjlB6-zLfzofF-)

[ISSUE_KEY]: AWaHMZUjlB6-zLfzofF-

Reply:
Not just `equals` but also `hashCode` and some tests?